### PR TITLE
{lib}[foss/2022a] CLIP v20230220 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node250.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2650 v4 @ 2.20GHz, 1 x NVIDIA Tesla P100-PCIE-16GB, 515.48.07, Python 3.6.8
See https://gist.github.com/smoors/f260157f497f18cd6d47dba6e1dfc856 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux Rocky Linux 8.6, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/9f8ec061d6515d78b6d08c94a833e095 for a full test report.

File: easybuild/easyconfigs/c/CLIP/CLIP-20230220-foss-2022a-CUDA-11.7.0.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+use_pip = True
+
+exts_list = [
+    ('ftfy', '6.1.1', {
+        'checksums': ['bfc2019f84fcd851419152320a6375604a0f1459c281b5b199b2cd0d2e727f8f'],
+    }),
+    (name, version, {
+        'source_urls': ['https://github.com/openai/CLIP/archive'],
+        'sources': [{'download_filename': 'a9b1bf5.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
```
Comment (Micket) [inline easybuild/easyconfigs/c/CLIP/CLIP-20230220-foss-2022a-CUDA-11.7.0.eb:34]:
```suggestion
        'sources': [{'download_filename': _commit + '.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
```

Comment (smoors) [reply easybuild/easyconfigs/c/CLIP/CLIP-20230220-foss-2022a-CUDA-11.7.0.eb]: that pesky `--inject-checksums`

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node250.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2650 v4 @ 2.20GHz, 1 x NVIDIA Tesla P100-PCIE-16GB, 515.48.07, Python 3.6.8
See https://gist.github.com/smoors/a33520501c6b400a043dae62027e8d27 for a full test report.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Going in, thanks @smoors!

**Pull request merged**